Formula/clusterawsadm.rb

```diff
@@ -0,0 +1,32 @@
+class Clusterawsadm < Formula
+  desc "Home for bootstrapping, AMI, EKS, and other helpers in Cluster API Provider AWS"
+  homepage "https://cluster-api-aws.sigs.k8s.io/clusterawsadm/clusterawsadm.html"
+  url "https://github.com/kubernetes-sigs/cluster-api-provider-aws.git",
+      tag:      "v2.0.2",
+      revision: "28bc9b8756d6d7f73038604038ddb3ccf4b22396"
+  license "Apache-2.0"
+  head "https://github.com/kubernetes-sigs/cluster-api-provider-aws.git", branch: "main"
+
+  livecheck do
+    url "https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases"
+    regex(%r{href=["']?[^"' >]*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+    strategy :page_match
+  end
+
+  depends_on "go" => :build
+
+  def install
+    # Don't dirty the git tree
+    rm_rf ".brew_home"
+
+    system "make", "clusterawsadm"
+    prefix.install "bin"
```

Comment:
I think this makes more sense.
```suggestion
    bin.install Dir["bin/*"]
```